query.py

```diff
@@ -143,7 +143,7 @@ def __call__(
 )
 _IGNORED_PREFIX_RE = r"^({0})\s*".format("|".join(_IGNORED_QUERY_PREFIXES))
 # Auto-capitalization corrections
-_CAPITALIZATION_REPLACEMENTS = (("í Dag", "í dag"),)
+_CAPITALIZATION_REPLACEMENTS = (("í Dag", "í dag"),("Á dagskrá", "á dagskrá"))
```

Comment:
What problem exactly is this addressing?

Reply:
This might be fixed in recent versions of GreynirPackage, I'll test and see, but the issue was simply that "á" was capitalized.

Reply:
Removed it, seems to be addressed in new GreynirPackage versions.